pages/common/nix-env.md

```diff
@@ -7,30 +7,30 @@
 
 `nix-env {{[-q|--query]}}`
 
-- Query installed packages:
+- Query installed packages (regex is supported):
```

Comment:
```suggestion
- Query installed packages (`regex` is supported):
```

Reply:
Style guide ikstructs to put technical wording in backticks and regex is technical wording in my opinion.